src/microfrontends/index.ts

```diff
@@ -71,6 +72,9 @@ export interface Hooks {
   useFeature?: typeof useFeature
 }
 
+export interface Utils {
+  getLocationPathName: typeof getLocationPathName
```

Comment:
I _think_ the practice is to set new things as optional to ensure MFEs don't need to mock things they don't need.
```suggestion
  getLocationPathName?: typeof getLocationPathName
```